### DISCUSSION TITLE
creating a table of data, need to make my own part or use list part?

### DISCUSSION THREAD

**Discussion by Eaglef90** [OrchardCMS/OrchardCore#12578] [Q&A]
I am trying to generate a tabled list of 3 columns, "Name, Date, Location". I was going to create my own Content Part to do this but then I saw a built in list part. Problem is I can't figure out how to apply that content part to what I want to do. I looked at the documentation and it makes me think this built in part is purly for listing content types (i.e. Blog Posts in a Blog site). Do I need to roll my own content part or am I missing something on how the list part works and if I am can someone tell me how to make it work for what I am looking for?

**Comment by ns8482e** [comment]
@Eaglef90 ListPart can contain other content items. Like content item blog has list part that can contain content items BlogPost. Here all content items are store separate documents.

Another part OC provides is BagPart that can also contain other content items, however here all content items are stored as single document

**Reply by Eaglef90** [reply]
Thanks, that is what I thought the ListPart was for but wanted to make sure before I spent the time to roll my own Content Part for this.

Or actually, maybe a bag part that contains 3 textfiles and then mutiple bag parts per conent type (page?, still trying to get the terminology down).
